src/styles/blocks/_body.scss

```diff
@@ -0,0 +1,4 @@
+body {
+  display: flex;
+  justify-content: center;
+}
```

Comment:
add `align-items: center` to center the calendarinside body
add `height: 100vh` to make body take all the space of your page 
read here -> https://www.w3schools.com/cssref/css_units.php